euphoria-hadoop/src/main/java/cz/seznam/euphoria/hadoop/output/HadoopSink.java

```diff
@@ -39,6 +43,11 @@
   private final SerializableWritable<Configuration> conf;
   private final SerializableWritable<JobID> jobID;
 
+  @GuardedBy("lock")
+  private final transient Map<TaskAttemptID, OutputFormat<K, V>> outputFormats = new HashMap<>();
+
+  private final transient Object lock = new Object();
```

Comment:
This has to cause NPE after deserialization and attempt to synchronize on the lock, because it will be `null`, because of being `transient.